projects/plugins/social/src/js/components/social-notes-toggle/index.tsx

```diff
@@ -29,11 +29,12 @@ const handleStateUpdating = ( updateFunction, updatingStateSetter, ...args ) =>
 };
 
 const SocialNotesToggle: React.FC< SocialNotesToggleProps > = ( { disabled } ) => {
-	const { isEnabled, notesConfig } = useSelect( select => {
+	const { isEnabled, notesConfig, newNoteUrl } = useSelect( select => {
 		const store = select( socialStore );
 		return {
 			isEnabled: store.isSocialNotesEnabled(),
 			notesConfig: store.getSocialNotesConfig(),
+			newNoteUrl: `${ store.getAdminUrl() }post-new.php?post_type=jetpack-social-note`,
```

Comment:
I wish Jetpack had this kind of stuff standardized. We discussed the same here p1713422972818109-slack-CDLH4C1UZ